recipes/openexr/all/CMakeLists.txt.patch

```diff
@@ -0,0 +1,18 @@
+--- CMakeLists.txt.orig 2018-08-10 10:34:26.000000000 +0200
++++ CMakeLists.txt
+@@ -13,6 +13,14 @@
+ set(OPENEXR_SOVERSION 24)
+
+ project(OpenEXR VERSION ${OPENEXR_VERSION})
++include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
++conan_basic_setup(KEEP_RPATHS)
```

Comment:
Why is this KEEP_RPATHS needed?

Reply:
@ericLemanissier ping

Reply:
I don't think it is necessary anymore. Could you please give it a go removing it? I think it worths a try. Thanks!